### ISSUE TITLE
Tankok, laktanya nélkül.

### ISSUE THREAD

**Issue by feqolosi** [akarnokd/open-ig#936]
Megtámadom az ellenség bolygóját; három laktanya és baromi sok tank van. Kilövöm a laktanyákokat, a tankjai kilőnek engem. Jövök másodjára, nincs laktanya, de mégis tonnaszámra van ellenséges tank. Ha a bolygó tankkapacitása elveszik, akkor a tankoknak nem kéne visszakerülnie a raktárba?

**Comment by Kriti83** [comment]
Következetlen, de így volt eredetiben is.

**Comment by Pistapapa** [comment]
Nem hiszem amúgy, hogy ez megállja a helyét. Attól hogy kilőtted felőle a laktanyát a tank még ott marad. Ennyi erővel akkor ha kilövöd a laktanyát akkor ami már nem fér el, az a csatában is felrobbanhatna rögtön. :D 
Szerintem ez nem hiba.

**Comment by TCH68k** [comment]
@Pistapapa: Nem, csata közben nyilván nem mennek sehova, de a csata végeztével, ha nincs hol tárolni őket, akkor nincs hol tárolni őket. Ennyi erővel akár laktanya nélkül is le lehetne szórni hatvan tonna tankot a bolygókra.
